packages/sdk/src/errors/error-info.ts

```diff
@@ -0,0 +1,34 @@
+import {BigNumberish} from 'ethers'
+
+export enum HolographErrorCode {
+  HOLO_SDK_10000 = 'HOLO_SDK_10000',
+  HOLO_SDK_10001 = 'HOLO_SDK_10001',
+  HOLO_SDK_10002 = 'HOLO_SDK_10002',
+  HOLO_SDK_10003 = 'HOLO_SDK_10003',
+}
+
+export const normalizeException = (err: any): Error => {
+  if (!(err instanceof Error)) {
+    err = new Error(err)
+  }
+  return err
+}
+
+export const errorInfo = {
+  [HolographErrorCode.HOLO_SDK_10000]: () => ({
+    message: `Unknown error.`,
+    description: 'An error of unknown source was identified',
+  }),
+  [HolographErrorCode.HOLO_SDK_10001]: (chainId: BigNumberish) => ({
+    message: `Failed to get network to chainId ${chainId}.`,
```

Comment:
maybe the word `for` would be better, not sure tho, like:

`Failed to get network for chainId ${chainId}.`

Reply:
@feliperodriguess you are right, it should be `Failed to get network for chain id ${chainId}`